app/assets/stylesheets/govuk-component/_govspeak.scss

```diff
@@ -8,6 +8,12 @@
     text-align: start;
   }
 
+  &.rich-govspeak {
+    strong {
+      font-weight: bold;
```

Comment:
Shouldn't this be enabled by this commit https://github.com/alphagov/static/commit/2184c82e45c186c71f12b026209eca610f816228?